src/libm-tester/tester2simddp.c

```diff
@@ -88,16 +88,12 @@ typedef Sleef___m512d_2 vdouble2;
 typedef Sleef___m512_2 vfloat2;
 #endif
 
-#ifdef ENABLE_VECEXT
-#define CONFIG 1
-#include "helpervecext.h"
-#include "norename.h"
-#endif
-
-#ifdef ENABLE_PUREC
-#define CONFIG 1
-#include "helperpurec.h"
-#include "norename.h"
+#ifdef ENABLE_AVX512FNOFMA
```

Comment:
The changes in this file seem to be unnecessary. If I understand correctly, these are the "nofma" and "pure_c*_scalar" target that are needed for the "determinism" patch, and are unrelated to the improvements of the payne hanek algorithm.

In fact, "rename*nofma.h"  is not included in this patch.

May I ask you to revert the changes in this file?